feat(UI): remove unnecessary spacings on the Profile page

zetok opened the pull request:
With corrected window title of own profile it's quite clear what the
page is about. Aside from that, removed elements didn't help at all to
explain what the page is about.

![spectacle ad3738](https://cloud.githubusercontent.com/assets/3148759/21290079/68c454b8-c4a5-11e6-881c-4778c1edf8cb.png)
↓
![spectacle aq3171](https://cloud.githubusercontent.com/assets/3148759/21290080/6db04ca2-c4a5-11e6-8977-d03ff907033c.png)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/3977)
<!-- Reviewable:end -->

zetok committed:
feat(UI): remove unnecessary spacings on the Profile page

Moved "header" under the scrollbar.

Renamed `Profile` and `User Profile` to `My profile`.

There is a regression with profile page header title no longer being
bold; according it Qt Creator it should be bold. No idea how to fix
that.

Thanks to @Diadlo for the code removing unnecessary header.

sudden6 commented:
I like it, other opinions? <img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/> 

---

Reviewed 1 of 2 files at r1.
Review status: 1 of 2 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3977#-:-KZDwBnBBRWOx5L8Q2cS:b-he3tl0)*
<!-- Sent from Reviewable.io -->

Diadlo commented:
Reviewed 1 of 2 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

*[src/widget/widget.cpp, line 1314 at r1](https://reviewable.io:443/reviews/qtox/qtox/3977#-KZG2B-wuKewqlI-M67S:-KZG2B-xViGHIqayfbMU:b-i3k43w) ([raw file](https://github.com/qtox/qtox/blob/4274537a43e9afc79ee7c74951da070b14a3cf04/src/widget/widget.cpp#L1314)):*
> ```cpp
>             return tr("Settings", "title of the window");
>         case ProfileDialog:
>             return tr("My Profile", "title of the window");
> ```

Make `P` lowercase

---

*[src/widget/form/profileform.cpp, line 58 at r1](https://reviewable.io:443/reviews/qtox/qtox/3977#-KZG2c63gVxCFBAf3u-U:-KZG2c64zRg3N0Fo62zO:bmfgxrq) ([raw file](https://github.com/qtox/qtox/blob/b392e2c320800c38f44370dd7deee53e7497b167/src/widget/form/profileform.cpp#L58)):*
> ```cpp
>     core = Core::getInstance();
> 
>     head = new QWidget(this);
> ```

I think it may also be removed

---

*[src/widget/form/profileform.cpp, line 65 at r1](https://reviewable.io:443/reviews/qtox/qtox/3977#-KZG2RH3ypw59NNtof-C:-KZG2RH4EcaxsDsQCwGN:b-mvxpem) ([raw file](https://github.com/qtox/qtox/blob/b392e2c320800c38f44370dd7deee53e7497b167/src/widget/form/profileform.cpp#L65)):*
> ```cpp
>     headLayout->addWidget(imgLabel);
> 
>     nameLabel = new QLabel();
> ```

Please, remove `nameLabel` from `.h` file

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3977)*
<!-- Sent from Reviewable.io -->

zetok commented:
As requested on IRC, I've restored the header, but without the unnecessary icon.

---

Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions, some commit checks pending.

---

*[src/widget/widget.cpp, line 1314 at r1](https://reviewable.io:443/reviews/qtox/qtox/3977#-KZG2B-wuKewqlI-M67S:-KZHMOSP4DwCRfs3FdPK:b-896fix) ([raw file](https://github.com/qtox/qtox/blob/4274537a43e9afc79ee7c74951da070b14a3cf04/src/widget/widget.cpp#L1314)):*
<details><summary><i>Previously, Diadlo (Polshakov Dmitry) wrote…</i></summary><blockquote>

Make `P` lowercase
</blockquote></details>

Done.

---

*[src/widget/form/profileform.cpp, line 58 at r1](https://reviewable.io:443/reviews/qtox/qtox/3977#-KZG2c63gVxCFBAf3u-U:-KZHIU-Ok6JgvGAkGYyJ:bmh7gkp) ([raw file](https://github.com/qtox/qtox/blob/b392e2c320800c38f44370dd7deee53e7497b167/src/widget/form/profileform.cpp#L58)):*
<details><summary><i>Previously, Diadlo (Polshakov Dmitry) wrote…</i></summary><blockquote>

I think it may also be removed
</blockquote></details>

Dunno about that – when I removed it, qTox segfaulted when I've tried to open profile.

---

*[src/widget/form/profileform.cpp, line 65 at r1](https://reviewable.io:443/reviews/qtox/qtox/3977#-KZG2RH3ypw59NNtof-C:-KZHNpbSp-UzViOhUOMy:b-a2ujfl) ([raw file](https://github.com/qtox/qtox/blob/b392e2c320800c38f44370dd7deee53e7497b167/src/widget/form/profileform.cpp#L65)):*
<details><summary><i>Previously, Diadlo (Polshakov Dmitry) wrote…</i></summary><blockquote>

Please, remove `nameLabel` from `.h` file
</blockquote></details>

Changed.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3977#-:-KZHMP9qBqlCbkT4zWGL:bvqownq)*
<!-- Sent from Reviewable.io -->

Diadlo commented:
IMO, it's worse than the two in your post (with old header and without header at all)
![spectacle x16329](https://cloud.githubusercontent.com/assets/6306608/21295341/77672cb2-c56c-11e6-9277-d2652e7ae406.png)

zetok commented:
It's similar to how current tab settings look like.

sudden6 commented:
@zetok I also liked your first version more

Diadlo commented:
```
[00:25] <zetok> right now scrollbar is only for the "content" of the page
[00:26] <Diadlo> Do you want to place header in "content"?
[00:26] <zetok> original "header" should be restored, and moved to the "content", so that it could be scrolled away
```

zetok commented:
Header moved under scrollbar.

Edit: 
![spectacle iv7718](https://cloud.githubusercontent.com/assets/3148759/21861957/1a058d9a-d82f-11e6-9753-ff6dddb25560.png)

sudden6 commented:
Reviewed 5 of 5 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3977)*
<!-- Sent from Reviewable.io -->
